src/sql/workbench/contrib/dashboard/browser/containers/dashboardFullSizeWidgetContainer.css

```diff
@@ -0,0 +1,18 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+dashboard-full-size-widget-container,
+.full-size-widget-container {
+	height: 100%;
```

Comment:
Why not just allow specifying the height and width of the widget as 100%? In general I think this kind of layout thing is better done as configuration on the actual components themselves rather than wrapping them in yet another container. 

At the very least this should at least be something more general like "LayoutContainer" that can be extended to support other types of layouts in the future - but first I think it's worth trying to get the widgets themselves to support being resized appropriately.